tutorial/game/tl/spanish/tutorial_screen_displayables.rpy

```diff
@@ -653,31 +653,31 @@ translate spanish strings:
 
     # tutorial_screen_displayables.rpy:9
     old "Text."
-    new "Text."
+    new "Texto."
 
     # tutorial_screen_displayables.rpy:9
     old "Boxes and other layouts."
-    new "Boxes y otros layouts."
+    new "Cajas y otros desplazadores."
```

Comment:
I think it would be good to replace the word 'desplazadores' with 'disposiciones'.